source/php-libraries.txt

```diff
@@ -160,3 +136,9 @@ Miscellaneous Projects
 - `PeclMongoPhpDoc <https://github.com/localgod/PeclMongoPhpDoc>`_ provides
   skeleton classes for the ``mongo`` extension, which may be used to support
   autocomplete and inline documentation for IDEs.
+
+- `PHPfastcache <https://github.com/PHPSocialNetwork/phpfastcache>`_ provides a simple
+high-performance backend cache system for MongoDB.
```

Comment:
Bullet points that span multiple lines need to match indentation, similar to the formatting of the previous bullet point.

```suggestion
- `PHPfastcache <https://github.com/PHPSocialNetwork/phpfastcache>`_ provides a simple,
  high-performance backend cache system for MongoDB.
```